libs/blocks/table/table.js

```diff
@@ -531,8 +531,17 @@ function applyStylesBasedOnScreenSize(table, originTable) {
       }
 
       rows.forEach((row) => {
+        const firstFilterCol = row.querySelector(`.col-${filters[0] + 1}`);
         const secondFilterCol = row.querySelector(`.col-${filters[1] + 1}`);
-        if (secondFilterCol) secondFilterCol.classList.add('force-last');
+
+        if (firstFilterCol && firstFilterCol.classList.contains('col-heading')) {
```

Comment:
Is optional chaining not an option?
```suggestion
        if (firstFilterCol?.classList.contains('col-heading')) {
```
Same for L541